app/src/main/java/nl/tudelft/pixelperfect/client/ClientListener.java

```diff
@@ -5,14 +5,19 @@
 import com.jme3.network.MessageListener;
 
 /**
+ * The ClientListenere waits for incoming messages from the server and interps them.
+ *
  * @author Jesse Tilro
+ * @author Floris Doolaard
  */
 public class ClientListener implements MessageListener<Client> {
     public void messageReceived(Client source, Message message) {
         if (message instanceof HelloMessage) {
```

Comment:
Maybe we should remove the entire HelloMessage in both client and server, it was just temporary for testing purposes.